package.json

```diff
@@ -25,6 +25,8 @@
   "scripts": {
     "prepare": "npm update --dev vl-ui-util && npm run prepare:util",
     "prepare:util": "npm explore vl-ui-util -- npm run util",
+    "performance:generate": "node ./performance/generate.js",
+    "performance:run": "npm run performance:generate && concurrently --kill-others \"npm:demo\" \"node ./performance/run.js\"",
     "test": "npm run test:unit && npm run test:e2e",
```

Comment:
test:performance bij test steken?

Reply:
Dat is eender voor mij. Als de rest dat ook wil, dan zal ik dat doen.

Reply:
Ik heb hierover geen mening.

Reply:
Het zijn testen, dus lijkt me inderdaad kleine verbetering.

Reply:
Done